matchms/filtering/harmonize_undefined_inchi.py

```diff
@@ -1,8 +1,19 @@
+from typing import List
 from ..typing import SpectrumType
 
 
-def harmonize_undefined_inchi(spectrum_in: SpectrumType, undefined="", aliases=None):
+def harmonize_undefined_inchi(spectrum_in: SpectrumType, undefined: str = "",
+                              aliases: List[str] = None) -> SpectrumType:
+    """Replace all aliases for empty/undefined inchi entries by ``undefined``.
```

Comment:
Explaining undefined confuses me, try to make it more clear.
```suggestion
    """Replace all aliases for empty/undefined inchi entries by value of ``undefined`` argument.
```

Reply:
Looks better indeed.